docker/single-process/docker-compose.yml

```diff
@@ -1,35 +1,39 @@
-mysqldata:
-  image: mysql:5.7
-  command: /bin/true
+# This needs at least compose 1.6.0
+version: '2'
 
-mysql:
-  image: mysql:5.7
-  volumes_from:
-    - mysqldata
-  environment:
-    MYSQL_ROOT_PASSWORD: myrootpassword
-    MYSQL_DATABASE: huginn
-    MYSQL_USER: huginn
-    MYSQL_PASSWORD: myhuginnpassword
+services:
+  mysqldata:
+    image: mysql:5.7
+    command: /bin/true
 
-huginn_web:
-  image: cantino/huginn-single-process
-  restart: always
-  extends:
-    file: environment.yml
-    service: huginn_base
-  ports:
-    - 3000:3000
-  links:
-    - mysql
+  mysql:
+    image: mysql:5.7
+    volumes_from:
+      - mysqldata
+    environment:
+      MYSQL_ROOT_PASSWORD: myrootpassword
+      MYSQL_DATABASE: huginn
+      MYSQL_USER: huginn
+      MYSQL_PASSWORD: myhuginnpassword
 
-huginn_threaded:
-  image: cantino/huginn-single-process
-  restart: always
-  extends:
-    file: environment.yml
-    service: huginn_base
-  links:
-    - mysql
-  command: /scripts/init bin/threaded.rb
+  huginn_web:
+    image: cantino/huginn-single-process
+    restart: always
+    extends:
+      file: environment.yml
+      service: huginn_base
+    ports:
```

Comment:
compose does not set the link environment variables automatically anymore, we need to add them for all huginn processes:
```
    environment:
      MYSQL_PORT_3306_TCP_ADDR: mysql
      MYSQL_PORT_3306_TCP_PORT: 3306
```